docs/plainmonthday.md

```diff
@@ -89,41 +89,43 @@ The `overflow` option is ignored if `thing` is a string.
 Example usage:
 
 ```javascript
-md = Temporal.PlainMonthDay.from('08-24'); // => 08-24
-md = Temporal.PlainMonthDay.from('2006-08-24'); // => 08-24
-md = Temporal.PlainMonthDay.from('2006-08-24T15:43:27'); // => 08-24
-md = Temporal.PlainMonthDay.from('2006-08-24T15:43:27Z'); // => 08-24
+md = Temporal.PlainMonthDay.from('08-24'); // => '08-24'
+md = Temporal.PlainMonthDay.from('2006-08-24'); // => '08-24'
+md = Temporal.PlainMonthDay.from('2006-08-24T15:43:27'); // => '08-24'
+md = Temporal.PlainMonthDay.from('2006-08-24T15:43:27Z'); // => '08-24'
 md = Temporal.PlainMonthDay.from('2006-08-24T15:43:27+01:00[Europe/Brussels]');
-// => 08-24
-md === Temporal.PlainMonthDay.from(md); // => true
+// => '08-24'
+md.equals(Temporal.PlainMonthDay.from(md)); // => true
```

Comment:
Same here as for PlainDate, and for the corresponding example in other types' documentation